### PR TITLE
[dt-overlay-at91][PATCH]

### PR DESCRIPTION
**Pull request by cybdarren**
Added Thermo5 click board support for sama5d27_som1_ek and sam9x60ek

### CODE REVIEW

File: sama5d27_som1_ek.its [hunk 1]
```diff
@@ -178,5 +194,11 @@
 			description = "FDT overlay blob for ov5640 sensor";
 			fdt = "fdt_ov5640";
 		};
+
+                thermo5 {
+                        description = "FDY overlay for Thermo5 click";
```
Comment (cristibirsan) [inline sama5d27_som1_ek.its]: You have a typo here FDY -> FDT

File: sama5d27_som1_ek.its [hunk 2]
```diff
@@ -134,6 +134,22 @@
...
+                        compression = "none";
+                        load = <0x23190000>;
+                        hash@1 {
+                                algo = "crc32";
+                        };
+                        hash@2 {
+                                algo = "sha1";
+                        };
+                };
+
```
Comment (cristibirsan) [inline sama5d27_som1_ek.its]: No need for space here

File: sama5d27_som1_ek/sama5d27_som1_ek_thermo5.dtso [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/dts-v1/;
```
Comment (cristibirsan) [inline sama5d27_som1_ek/sama5d27_som1_ek_thermo5.dtso:11]: Add license information here. We use SPDX-License-Identifier: (GPL-2.0+ OR MIT)

**Review by cristibirsan — CHANGES_REQUESTED** [review]
Thank you for sending this and congratulations for your first patches !

There are some minor fixes needed before merging. I added them in the review.

Regards,
Cristian

File: sama5d27_som1_ek.its (hunk 2 above)
Comment (cybdarren) [reply sama5d27_som1_ek.its]: Space removed to be consistent with other files

File: sama5d27_som1_ek.its (hunk 1 above)
Comment (cybdarren) [reply sama5d27_som1_ek.its]: Typo corrected

File: sama5d27_som1_ek/sama5d27_som1_ek_thermo5.dtso (hunk 3 above)
Comment (cybdarren) [reply sama5d27_som1_ek/sama5d27_som1_ek_thermo5.dtso:11]: License headers added to thermo5 clicks

**Comment by cybdarren** [comment]
Requested changes made…

Cheers,
   Darren

**Comment by cristibirsan** [comment]
Merged. Thank you,
Cristian

**Pull request closed without merging**